packages/react/fast-components-react-base/src/stack-panel/stack-panel.tsx

```diff
@@ -0,0 +1,580 @@
+import { isArray, isFunction } from "util";
```

Comment:
"util" looks like the wrong import source - that is a nodejs module. Did you mean "lodash-es"?

Reply:
done